From: Markus Koschany <apo@gambaru.de>
Date: Sat, 19 Jan 2013 09:29:01 +0100
Subject: no donators menu

The patch was taken from http://pkgs.fedoraproject.org/cgit/xarchiver.git/tree/
and modified so that the donate button leads the user to xarchiver's
actual sourceforge.net presence.
---
 src/interface.c |   20 ++++++--------------
 src/interface.h |    4 ++--
 2 files changed, 8 insertions(+), 16 deletions(-)

diff --git a/src/interface.c b/src/interface.c
index 852b9a7..6efc089 100644
--- a/src/interface.c
+++ b/src/interface.c
@@ -310,22 +310,14 @@ void xa_create_main_window (GtkWidget *xa_main_window,gboolean show_location,gbo
 	gtk_widget_show (tmp_image);
 	gtk_image_menu_item_set_image (GTK_IMAGE_MENU_ITEM (help1),tmp_image);
 
+	donate = gtk_image_menu_item_new_with_mnemonic (_("_Donate"));
+	gtk_widget_show (donate);
+	gtk_container_add (GTK_CONTAINER (menuitem4_menu),donate);
+
 	about1 = gtk_image_menu_item_new_from_stock ("gtk-about",accel_group);
 	gtk_widget_show (about1);
 	gtk_container_add (GTK_CONTAINER (menuitem4_menu),about1);
 
-	thanks_to = gtk_image_menu_item_new_with_mnemonic (_("_Thanks to"));
-	gtk_widget_show (thanks_to);
-	gtk_container_add (GTK_CONTAINER (menuitem4_menu),thanks_to);
-
-	thanks_to_submenu = gtk_menu_new();
-	gtk_widget_show (thanks_to_submenu);
-	gtk_menu_item_set_submenu (GTK_MENU_ITEM (thanks_to),thanks_to_submenu);
-
-	donators = gtk_image_menu_item_new_with_mnemonic ("Your name and website here");
-	gtk_widget_show (donators);
-	gtk_container_add (GTK_CONTAINER (thanks_to_submenu),donators);
-
 	/* Create the toolbar */
 	toolbar1 = gtk_toolbar_new ();
 	gtk_widget_show (toolbar1);
@@ -543,7 +535,7 @@ void xa_create_main_window (GtkWidget *xa_main_window,gboolean show_location,gbo
 	g_signal_connect ((gpointer) multi_extract_menu,"activate",G_CALLBACK (xa_show_multi_extract_dialog),NULL);
 	g_signal_connect ((gpointer) help1,"activate",G_CALLBACK (xa_show_help),NULL);
 	g_signal_connect ((gpointer) about1,"activate",G_CALLBACK (xa_about),NULL);
-	g_signal_connect ((gpointer) donators,"activate",G_CALLBACK (xa_show_donate_page_on_the_web),NULL);
+	g_signal_connect ((gpointer) donate,"activate",G_CALLBACK (xa_show_donate_page_on_the_web),NULL);
 
 	g_signal_connect ((gpointer) New_button,	"clicked",G_CALLBACK (xa_new_archive),NULL);
 	g_signal_connect ((gpointer) Open_button,	"clicked",G_CALLBACK (xa_open_archive),NULL);
@@ -826,7 +818,7 @@ gchar *xa_create_password_dialog(XArchive *archive)
 
 void xa_show_donate_page_on_the_web(GtkMenuItem *item,gpointer data)
 {
-	xa_activate_link (NULL,"http://xarchiver.xfce.org/contribute.html",NULL);
+	xa_activate_link (NULL,"http://xarchiver.sourceforge.net/contribute.html",NULL);
 }
 
 void xa_create_popup_menu()
diff --git a/src/interface.h b/src/interface.h
index c8e7fa0..c58c0c0 100644
--- a/src/interface.h
+++ b/src/interface.h
@@ -48,8 +48,8 @@ GtkWidget *xa_popup_menu, *xa_main_window, *vbox1, *hbox_sb, *menubar1, *menuite
 *name_label, *type_label, *compression_data, *encrypted_data, *encrypted_label,*number_of_files_data, *content_data, *comment_data, *size_data, *modified_data, *path_data, *type_data,
 *name_data, *separatormenuitem1, *separatormenuitem2, *separatormenuitem3, *separatormenuitem4, *separatormenuitem5, *separatormenuitem6, *quit1,
 *close1, *check_menu, *properties, *menuitem2, *menuitem2_menu, *addfile, *extract_menu, *delete_menu, *comment_menu, *multi_extract_menu,*view_shell_output1,
-*prefs_menu,*password_entry_menu, *image1, *image2, *menuitem4, *thanks_to,*thanks_to_submenu,*select_all,*deselect_all,*select_pattern, *exe_menu, *menuitem4_menu,
-*about1, *help1, *donators,*toolbar1, *toolbar2, *hbox1, *tmp_image, *pad_image, *New_button, *Open_button, *back_button, *home_button, *forward_button,
+*prefs_menu,*password_entry_menu, *image1, *image2, *menuitem4, *donate,*select_all,*deselect_all,*select_pattern, *exe_menu, *menuitem4_menu,
+*about1, *help1,*toolbar1, *toolbar2, *hbox1, *tmp_image, *pad_image, *New_button, *Open_button, *back_button, *home_button, *forward_button,
 *up_button,*separatortoolitem1,*separatortoolitem2,*separatortoolitem3,*AddFile_button,*Extract_button,*Stop_button,*toolitem1,*location_label,
 *location_entry,*hpaned1,*archive_dir_treeview,*scrolledwindow2,*ddelete,*rename_menu,*rrename,*cut,*copy,*paste,*view,*open_popupmenu;
 
